pkg/registry/resource/resourceslice/strategy_test.go

```diff
@@ -206,7 +218,27 @@ func TestResourceSliceStrategyCreate(t *testing.T) {
 			partitionableDevices: true,
 			expectObj: func() *resource.ResourceSlice {
 				obj := sliceWithPartitionableDevices.DeepCopy()
-				obj.ObjectMeta.Generation = 1
+				obj.Generation = 1
+				return obj
+			}(),
+		},
+		"drop-fields-binding-conditions": {
```

Comment:
For the sake of completeness let's also cover the two combinations where only one feature is enabled.